python/ray/experimental/workflow/tests/test_fork_join.py

```diff
@@ -0,0 +1,38 @@
+import ray.experimental.workflow as rayflow
```

Comment:
maybe just call it workflow? it looks a little big wired later it's rayflow, but the exposed API is workflow

Reply:
sure